<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Math 24</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/widgetIcon.jpg</normaloff>:/images/widgetIcon.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{background-image: url(:/images/background.jpg);}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>0</y>
     <width>781</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#localTab{background-image: url(:/images/background.jpg);}
#netTab{background-image: url(:/images/background.jpg);}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="localTab">
    <attribute name="title">
     <string>Local</string>
    </attribute>
    <widget class="QLineEdit" name="input1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Input1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Input2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Input3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Input4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input from file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileBrowseButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputBrowser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>531</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="retryButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Retry</string>
     </property>
    </widget>
    <widget class="QPushButton" name="historyButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>480</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>History...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Math 24</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calcButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>90</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculate!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="netTab">
    <attribute name="title">
     <string>Online</string>
    </attribute>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Timer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min(s)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sec(s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="timerClearButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>20</width>
       <height>521</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="readyButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="calcResBrowser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="combatResBrowser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>421</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combat Result</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchGameButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search for game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hostGameButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Host a game</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
